### PR TITLE
core(stacks): actually support joomla and october cms

### PR DESCRIPTION
**Pull request by connorjclark**
we needed these PRs https://github.com/GoogleChrome/lighthouse-stack-packs/pull/59 https://github.com/GoogleChrome/lighthouse-stack-packs/pull/60

and we needed to connect joomla detection to stack pack ...

### CODE REVIEW

File: lighthouse-core/test/lib/stack-packs-test.js [hunk 1]
```diff
@@ -22,11 +22,7 @@ describe('stack-packs lib', () => {
     const result = lighthouseStackPacksDep
       .filter(p => !stackPacksLib.stackPacksToInclude.find(p2 => p2.packId === p.id))
       .map(p => p.id);
-    expect(result).toMatchInlineSnapshot(`
-      Array [
-        "joomla",
```
Comment (brendankenny) [inline lighthouse-core/test/lib/stack-packs-test.js:27]: lolll

Comment (connorjclark) [reply lighthouse-core/test/lib/stack-packs-test.js:27]: ya it didnt work for its intended purpose lol

Comment (brendankenny) [reply lighthouse-core/test/lib/stack-packs-test.js:27]: well at least we know it works :)

Comment (patrickhulce) [reply lighthouse-core/test/lib/stack-packs-test.js:27]: can we convert this to `toEqual([]);` with a comment that we shouldn't bother updating a stack pack until detectors are in place? :)

**Review by patrickhulce — APPROVED** [review]
change itself is fine, test improvements though :)

**Pull request merged**